lib/entities/entities.js

```diff
@@ -16,26 +17,28 @@ const { EVENTS } = require("../api_definitions");
 class Entities extends EventEmitter {
   #storage;
   #dataPath;
+  #configFileName;
 
   constructor(id) {
     super();
 
     this.id = id;
     this.#storage = {};
-    this.#dataPath = `${this.id}.json`;
+    this.#dataPath = process.env.UC_CONFIG_HOME;
+    this.#configFileName = path.join(this.#dataPath, `${this.id}_entities.json`);
```

Comment:
With the changed configuration file name (appended `_entities`): won't this make all integrations loose their configured entities, and users have to reconfigure everything?
Or isn't this feature used in the available integrations?

At least the simulated-light example will loose the configured entities!

Reply:
As far as I know nothing was using this, not to mention that because it was not working (file couldn't be saved) I doubt it breaks anything.

Reply:
The file was saved in external user integrations. If they rely on the configuration file, this change breaks them.
So this needs at least a **breaking change** note.

The simulated-light example automatically adds the configured entities again ( `uc.availableEntities.addEntity` calls), and the restored configuration doesn't seem to be required, but if an external integration relies on the configuration file, then they are lost.

I'm actually quite confused about the purpose of the `available.json` & `configured.json` files. Something is missing too: the `configured` file gets automatically saved whenever the core subscribes (`IntegrationAPI#subscribeEvents(entities)`) or unsubscribes (`IntegrationAPI#unSubscribeEvents(entities)`) an entity, but the `available` file is never saved. But at startup there's always a log message: `ENTITIES(available): No storage file, will create one later: available.json`.

If the configuration file functionality has a real purpose, it needs to be documented on how to use it. Otherwise it should be removed and it's up to the integrations to persist data if they require to.

Reply:
Can we resolve this PR?
I still have no idea for what the two configuration files are useful. Either we remove them completely, or there needs to be clear documentation on how they can be used.